### PR TITLE
Make clippy happy

### PR DESCRIPTION
**Pull request by 0xPoe**
See the commits.

### CODE REVIEW

File: src/cli/common.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@ use std::fmt::Display;
 use std::fs;
 use std::io::{BufRead, ErrorKind, Write};
 use std::path::Path;
-use std::sync::Arc;
+use std::rc::Rc;
```
Comment (rbtcollins) [inline src/cli/common.rs]: I think this one will get resolved as we move further into async. I think we should just permit it or ignore it if its not stabilised.

File: src/config.rs [hunk 2]
```diff
@@ -3,8 +3,8 @@ use std::fmt::{self, Display};
 use std::io;
 use std::path::{Path, PathBuf};
 use std::process::Command;
+use std::rc::Rc;
 use std::str::FromStr;
-use std::sync::Arc;
```
Comment (rbtcollins) [inline src/config.rs]: ditto

File: src/dist/manifestation/tests.rs [hunk 3]
```diff
@@ -2079,7 +2080,7 @@ fn reuse_downloaded_file() {
                           temp_cfg| {
         prevent_installation(prefix);
 
-        let reuse_notification_fired = Arc::new(Cell::new(false));
+        let reuse_notification_fired = Rc::new(Cell::new(false));
```
Comment (rbtcollins) [inline src/dist/manifestation/tests.rs]: ditto

File: src/cli/common.rs (hunk 1 above)
Comment (0xPoe) [reply src/cli/common.rs]: Updated. Make sense.

**Pull request merged**